docs/advanced-recipes/atom-creators.mdx

```diff
@@ -291,3 +291,61 @@ function EvenCounter() {
   return <>Count was set to an even number {evenCount} times.</>
 }
 ```
+
+## atomWithBroadcast
+
+> `atomWithBroadcast` creates an atom. The atom will be shared between
+> browser tabs and frames, similar to `atomWithStorage` but with the 
+> initialization limitation.
+
+This can be useful when you want the state to interact with each other without
+the use of localStorage and that by just using The Broadcast Channel API allows
+basic communication between browsing contexts (that is, windows, tabs, frames, 
+create a component or iframes) and workers on the same origin.
+
+```ts
+import { atom } from "jotai";
+
+export function atomWithBroadcast<Value>(key: string, initialValue: Value) {
+  const channel = new BroadcastChannel(key);
+
+
+  const baseAtom = atom(initialValue);
+
+  const broadcastAtom = atom<Value, { fromOnMessage: boolean; value: Value }>(
+    (get) => get(baseAtom),
+    (get, set, update) => {
+      if (!update.fromOnMessage) {
+        channel.postMessage(update.value);
+      }
+      set(baseAtom, update.value);
+    }
+  );
+  broadcastAtom.onMount = (setAtom) => {
+    channel.onmessage = (event) => {
+      setAtom({ fromOnMessage: true, value: event.data });
+    };
+  };
+  const returnedAtom = atom<Value, Value>(
+    (get) => get(broadcastAtom),
+    (get, set, update) => {
+      set(broadcastAtom, { fromOnMessage: false, value: update });
+    }
+  );
+  return returnedAtom;
+}
+
+const atomBroad = atomWithBroadcast("count", 0);
```

Comment:
Let's follow the general naming of jotai's atoms! like something more meaningful + `atom` in the end. like `broadcastAtom`